theme/src/resources/templates/category_pages/liferay_dxp_7_3.hbs

```diff
@@ -28,9 +28,12 @@
 
 		const documentElementLang = document.documentElement.lang;
 
+		var alertChildren = null;
+
 		Liferay.render(
 			Liferay.ProductTabs,
 			{
+				alert: alertChildren,
```

Comment:
@cole-liferay 
If `alertChildren` is set as `null`, I don't think we need to include the alert here.